src/System.ComponentModel.Primitives/ref/System.ComponentModel.Primitives.cs

```diff
@@ -39,7 +39,7 @@ public partial class CategoryAttribute : System.Attribute
         public static System.ComponentModel.CategoryAttribute Layout { get { throw null; } }
         public static System.ComponentModel.CategoryAttribute Mouse { get { throw null; } }
         public static System.ComponentModel.CategoryAttribute WindowStyle { get { throw null; } }
-        public override bool Equals(object obj) { throw null; }
+        public override bool Equals(object? obj) { throw null; }
         public override int GetHashCode() { throw null; }
         protected virtual string GetLocalizedString(string value) { throw null; }
```

Comment:
GetLocalizedString should return a nullable string.

Reply:
I was thinking same at first, but then noticed the only method called here returning not null even suppresed possible null return, maybe that one annotated incorrectly? From SR.sc:
```csharp
internal static string GetResourceString(string resourceKey, string? defaultString = null)
{
    if (UsingResourceKeys())
    {
        return defaultString ?? resourceKey;
    }

    string? resourceString = null;
    try
    {
        resourceString = ResourceManager.GetString(resourceKey);
    }
    catch (MissingManifestResourceException) { }

    if (defaultString != null && resourceKey.Equals(resourceString))
    {
        return defaultString;
    }

    return resourceString!; // only null if missing resources
}
```

Reply:
That's the base implementation, but it's virtual, so derived types could be overriding it and returning null.  Since it's protected, really the only code that might call this method is this class (technically a derived one could as well, but that's unlikely), and it's already checking to see if the method returns null, which both suggests that was part of the design and means that there's little harm in allowing null.

Reply:
oh I see, good to know